docs/tutorials/timeseries/forecasting-indepth.md

```diff
@@ -12,7 +12,7 @@ This tutorial provides an in-depth overview of the time series forecasting capab
     - Hyperparameter tuning
     - Forecasting irregularly-sampled time series
 
-This tutorial assumes that you are familiar with the contents of the [basic tutorial](forecasting-quickstart.md).
+This tutorial assumes that you are familiar with the contents of the [basic tutorial](forecasting-quickstart.html).
```

Comment:
I think idiomatic way to reference other pages is to have a label in the target doc [see source](https://github.com/awslabs/autogluon/blob/master/docs/tutorials/cloud_fit_deploy/cloud-aws-sagemaker-training.md?plain=1#L2):
```
:label:`cloud_aws_sagemaker_fit`
```
And then you can [do a reference](https://github.com/awslabs/autogluon/blob/master/docs/tutorials/cloud_fit_deploy/cloud-aws-sagemaker-deployment.md?plain=1#L4):
```
:ref:`cloud_aws_sagemaker_fit`
```

Reply:
Thanks for the pointer!